src/commands/modcommands/unban.ts

```diff
@@ -13,7 +13,7 @@ const unban = async(member: GuildMember) => {
 
     await Ban.remove(member.user);
 
-    if (!restoreRoles(member)) return "Eine der angegebenen Rollen für das bannen existiert nich.";
+    if (!await restoreRoles(member)) return "Eine der angegebenen Rollen für das bannen existiert nich.";
```

Comment:
Das war ein lustiger bug, bei dem die condition immer false is, weil das promise immer true is.